### ISSUE TITLE
Hang with RtAudio/Asio/Voicemeeter with another asio device is active

### ISSUE THREAD

**Issue by oleg68** [GrandOrgue/grandorgue#1789]
If GrandOrgue uses `Rt: asio: Voicemeeter Virtual ASIO` device, and VoiceMeter uses another asio device (ex. ASIO4ALL), GrandOrgue hangs after exiting from the Settings dialog.

There are two possible workarounds:
- Use `RtAudio: ds: VoiceMeeter Input` instead of `RtAudio: asio: Voicemeeter Virtual ASIO`
- Use a WDM device instead of ASIO in the Voicemeeter settings

**Comment by oleg68** [comment]
@Psi58 could you test the workarounds with https://github.com/oleg68/GrandOrgue-official/tree/bugfix/crash-rtasio ?

**Comment by Psi58** [comment]
> @Psi58 could you test the workarounds with https://github.com/oleg68/GrandOrgue-official/tree/bugfix/crash-rtasio ?

@oleg68 Yes, I would be happy to. But (a) I am not a developer and (b) I don't have an available Linux or OS X environment (which, if I read BUILD.md correctly, I would need). Can you build the Windows executable, or advise how I can do that?
Many thanks for your help, I am sorry that I don't have the skill to work this out myself 😥

**Comment by oleg68** [comment]
@Psi58 Excuse me, I posted a wrong link. The right one is https://github.com/oleg68/GrandOrgue-official/actions/runs/7685099650

**Comment by Psi58** [comment]
Thanks @oleg68 . Both workarounds are good and stable.

**Comment by oleg68** [comment]
@Psi58 Thank you for your testing.

The poblem may be reformulated: GO hangs when two asio devices are active at the same time.

**Comment by oleg68** [comment]
The issue can be reproduced with ASIO4ALL.

The workaround: open the ASIO4ALL settings panel and exit from it.

**Comment by oleg68** [comment]
It is not reproducible with recent versions of VoiceMeter aand Asio4All
